docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Remove default providers and add OpenTelemetry logging provider
+builder.Logging.ClearProviders().AddOpenTelemetry(logging =>
+{
+    logging.IncludeScopes = true;
+
+    var resourceBuilder = ResourceBuilder
+        .CreateDefault()
+        .AddService(builder.Environment.ApplicationName);
+
+    logging.SetResourceBuilder(resourceBuilder)
+        .AddConsoleExporter();
```

Comment:
Sam suggested that we should consider OTLP + some local viewer tool.

Reply:
For this document - keep OTel ConsoleExporter, remove Console Provider, leave a comment with OTLP exporter, and mention in the README. Later when a backend tool (e.g. something like Jaeger) is ready, revisit the example and see if we should switch to OTLP + backend tool.

For the .NET official doc, use the default (Console Provider), link to OpenTelemetry doc (which will be added to .NET docs) for folks who want to export in other ways.

Reply:
I've made the update.